docs/reference-guides/block-api/block-supports.md

```diff
@@ -548,25 +549,41 @@ supports: {
     spacing: {
         margin: true,  // Enable margin UI control.
         padding: true, // Enable padding UI control.
+        blockGap: true,  // Enables blockGap UI control.
```

Comment:
Tiny thought: in the UI it's referred to as block spacing, should we reflect that here?

```suggestion
        blockGap: true,  // Enables block spacing UI control.
```

Reply:
Good point! Thanks for commit suggestion